images/tier1/grade-visible.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 21 18">
```

Comment:
The viewbox needs to be `0 0 18 18`. To get that to work though you'll need to add a transform that moves it `3px` and then SVGOMG can optimize it. I can give you a hand with fixing this.

Reply:
I've reached out to Nick to try and get this resolved, so ignore for now. :)